config/src/main/java/com/alibaba/nacos/config/server/utils/RegexParser.java

```diff
@@ -44,7 +42,7 @@ public static String regexFormat(String regex) {
         result.append("^");
         for (int i = 0; i < regex.length(); i++) {
             char ch = regex.charAt(i);
-            if (CharUtils.isAsciiAlphanumeric(ch) || CharUtils.isAsciiNumeric(ch)) {
+            if (isAsciiAlphanumeric(ch)) {
```

Comment:
Why not move them to CharUtils？

Reply:
Only used in this one place,So when I was writing the tool class, I had already planned to deal with it this way.

Reply:
OK